rainier-plot/src/main/scala/com/stripe/rainier/plot/EvilTraceplots.scala

```diff
@@ -0,0 +1,134 @@
+/*
+EvilTraceplots.scala
+
+Some diagnostic plots using EvilPlot
+
+ */
+
+package com.stripe.rainier.plot
+
+object EvilTraceplots {
```

Comment:
nit: for consistency with `EvilPlot`, I'd prefer `EvilTracePlot`

Reply:
Done.